js/src/product-feed/review-request/review-request-modal.js

```diff
@@ -22,7 +22,7 @@ const ReviewRequestModal = ( {
 } ) => {
```

Comment:
💅

I think it would be nice to add in the future some JSDocs so it is easier to understand some variables.